test/calculator.test.mjs

```diff
@@ -49,40 +52,43 @@ describe('calculator', function () {
     });
 
   xit('should display a number if number key is pressed', function () {
```

Comment:
I recommend changing all the `xit` to `it` in this file. It'll cause all your tests to fail, but will give you a clear idea of what needs to be fixed. Or you can comment out or delete tests you are not ready for and add them back when you are adding new functionality. If you run `npm run test` now it looks like everything is passing.

Reply:
That makes sense. When I run my test I like to run a single test at a time and xit out the others. But that wouldn't really be best practice for writing tests?